setup.cfg

```diff
@@ -1,13 +1,21 @@
+[metadata]
```

Comment:
The metadata is in pyproject.toml. Why was this needed on your end ?  It works without for me doing this:

(.venv) PS C:\sb\privat\flashcontainer> pip install .

Processing c:\sb\privat\flashcontainer
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
  Preparing metadata (pyproject.toml) ... done
Collecting json5>=0.9.10
  Using cached json5-0.9.10-py2.py3-none-any.whl (19 kB)
Collecting lxml>=4.9.2
  Using cached lxml-4.9.2-cp39-cp39-win_amd64.whl (3.9 MB)
Collecting crc>=3.0
  Downloading crc-4.1.0-py3-none-any.whl (7.6 kB)
Collecting intelhex>=2.3.0
  Using cached intelhex-2.3.0-py2.py3-none-any.whl (50 kB)
Building wheels for collected packages: flashcontainer
  Building wheel for flashcontainer (pyproject.toml) ... done
  Created wheel for flashcontainer: filename=flashcontainer-0.0.4-py3-none-any.whl size=28701 sha256=56658562f21c20daca620506e5983be3ac72cdeb133137608b087e23665ee060
  Stored in directory: C:\Users\schulz\AppData\Local\Temp\pip-ephem-wheel-cache-g6n0ogwh\wheels\96\0e\52\4af71bf0dc83d48be022ef1726e9f851a8f353e6c6f69c21c8
Successfully built flashcontainer
Installing collected packages: json5, intelhex, lxml, crc, flashcontainer
Successfully installed crc-4.1.0 flashcontainer-0.0.4 intelhex-2.3.0 json5-0.9.10 lxml-4.9.2

Reply:
Only pip uses it, but not local installation with python setup.yp install